docs/how-to-guides/adding-file-osds.md

```diff
@@ -0,0 +1,54 @@
+# Add virtual OSDs for test clusters
+
+MicroCeph also supports creating and using files as backing devices for OSDs. This is a handy feature for testing out MicroCeph without commiting physical block devices.
+
+OSDs are added by triggering `add-osd action` on a microceph unit.
+
+## Procedure
+
+### 1. Add file based OSDs to the microceph cluster
+
+       juju run microceph/0 add-osd loop-spec="4G,1"
+
+Multiple OSDs (assuming count N) can be added in the `add-osd` action.
+
+       juju run microceph/0 add-osd loop-spec="4G,<N>"
+
+The output of `add-osd action` should look similar to this:
+
+```
+$ juju run microceph/0 add-osd loop-spec="4G,3"
+Running operation 3 with 1 task
+  - task 4 on unit-microceph-0
+Waiting for task 4...
+status: success
+```
+
+### 2. Run ceph cluster status to check if OSDs are up
+
+
+       juju ssh microceph/leader sudo microceph.ceph status
```

Comment:
Nit: with snapstore installed microceph we have an alias for ceph, so this could be shortened to:
```
      juju ssh microceph/leader sudo ceph status
```